core/entity/patient.py

```diff
@@ -20,3 +21,15 @@ class Patient(BaseModel):
     created_user: Optional[Username] = None
     class Config():
         orm_mode = True
+
+class PatientSmall(BaseModel):
```

Comment:
Id, Name, Age, Contact Details, Gender, Date of Birth, Address and Created time is required.